app.py

```diff
@@ -3,6 +3,8 @@
 
 
 def main():
+    """Entry point for application"""
```

Comment:
Minor: `for the application`

Reply:
Changed to `Entry point for the application`

Reply:
You sure about that 
![image](https://user-images.githubusercontent.com/5748299/78312408-ae7cb380-7521-11ea-82eb-6ae43f0bf10c.png)